contracts/src/common/TENERC20.sol

```diff
@@ -2,11 +2,11 @@
 pragma solidity ^0.8.4;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "../messaging/IMessageBus.sol";
+import "../cross_chain_messaging/common/IMessageBus.sol";
 
-// This is an implementation of a canonical ERC20 as used in the Obscuro network
+// This is an implementation of a canonical ERC20 as used in the TEN network
 // where access to data has to be restricted.
-contract ObsERC20 is ERC20 {
+contract TENERC20 is ERC20 {
```

Comment:
aren't these for testing as well: TenERC20 and WrappedErc20?

Reply:
good shout - have moved these 2 and `ConstantSupplyERC20` into testing as well